src/dotnet/APIView/APIViewWeb/Helpers/CommonUtilities.cs

```diff
@@ -69,6 +71,127 @@ public static bool IsSDKLanguageOrTypeSpec(string language)
         }
     }
 
+    /// <summary>
+    /// Represents the plane classification of a package
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum PackageType
+    {
+        /// <summary>
+        /// Data plane package (client libraries for Azure services)
+        /// </summary>
+        Data,
+        
+        /// <summary>
+        /// Management plane package (resource management libraries)
+        /// </summary>
+        Management,
+        
+        /// <summary>
+        /// Cannot determine plane type from package name
+        /// </summary>
+        Unknown
+    }
+
+    /// <summary>
+    /// Package classification utilities for distinguishing data plane vs management plane packages
+    /// </summary>
+    public static class PackageHelper
+    {
+        /// <summary>
+        /// Classifies a package as Data, Management, or Unknown plane based on package name and language
+        /// </summary>
+        /// <param name="packageName">The package name to analyze</param>
+        /// <param name="language">The programming language (C#, Go, Python, Java, JavaScript)</param>
+        /// <returns>PackageType enum indicating Data, Management, or Unknown</returns>
+        public static PackageType ClassifyPackageType(string packageName, string language)
+        {
+            // Validate inputs - return Unknown
+            if (string.IsNullOrWhiteSpace(packageName) || string.IsNullOrWhiteSpace(language) || !LanguageHelper.IsSDKLanguage(language))
+            {
+                return PackageType.Unknown;
+            }
+
+            // Normalize language name for comparison
+            var normalizedLanguage = language.Trim().ToLowerInvariant();
+            
+            switch (normalizedLanguage)
+            {
+                case "c#":
+                    // Management: strings that match regex /^Azure\.ResourceManager\./
+                    if (packageName.StartsWith("Azure.ResourceManager.", StringComparison.Ordinal))
```

Comment:
We should not do this filtering here. what happens if management plane team decides to add a new prefix in the future?  They won't know that there is a dependency in APIView side for such namespace prefix change. Each package already has such info and that is available as part of package info in the pipeline. more than 95% of our API reviews are generated from pipeline. If this info is missing for manually created reviews, then just tag it as "Unknown" type. But I strongly recommend not doing this in APIView side.

FYI @maririos, @tjprescott, @AlitzelMendez